imperial_coldfront_plugin/gpfs_client.py

```diff
@@ -255,6 +255,35 @@ def create_fileset_directory(
                 return task_data
             raise
 
+    @check_job_status
```

Comment:
Not needed and causes an error as GET endpoints just return data and don't trigger the GPFS server to run a background job.